chain/bitcoind_events.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/lightninglabs/gozmq"
+	"github.com/mroth/jitter"
```

Comment:
How critical is it that the polling period be "merely inconsistent" vs highly entropic? If it's the later, have we validated which the package applies? I'm considering PRNG output looping etc

Reply:
I think we don't need that much entropy for intervals since it also depends on the range we choose. If we decide a jitter scale of 0.1 with an interval of 10s then we are jittering between 9 seconds and 11 seconds, not that much of a difference. But yeah we can always call `rand.Seed` to make it "more" randomized.